CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Welcome to Ceph NVMe-oF Gateway contributing guide <!-- omit in toc -->
+
+## New contributor guide
+
+The Ceph NVMe-oF Gateway project pivots around 2 other major Open Source projects:
+- [SPDK](https://spdk.io/) [[source code](https://github.com/spdk/spdk/)], whih internally relies on the DPDK project.
+- [Ceph](https://ceph.com/) [[source code](https://github.com/ceph/ceph)].
+
+## Reporting issues
+
+If you found an issue, please identify first whether the issue comes from the NVMe-oF Gateway, or any of the underlying components:
```

Comment:
s/found/find/

s/please identify first/identify/

Reply:
Thanks @zdover23! So from what I understand here, I should avoid 'please' in general, right? I'll remove those.